[2021-08-07 14:20:19,934] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: standardized_pipeline.create_standardized_table_orders 2021-08-07T14:20:18.742702+00:00 [queued]>
[2021-08-07 14:20:19,991] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: standardized_pipeline.create_standardized_table_orders 2021-08-07T14:20:18.742702+00:00 [queued]>
[2021-08-07 14:20:19,993] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 14:20:19,995] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-07 14:20:19,996] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 14:20:20,021] {taskinstance.py:1107} INFO - Executing <Task(MySqlOperator): create_standardized_table_orders> on 2021-08-07T14:20:18.742702+00:00
[2021-08-07 14:20:20,047] {standard_task_runner.py:52} INFO - Started process 4571 to run task
[2021-08-07 14:20:20,098] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'standardized_pipeline', 'create_standardized_table_orders', '2021-08-07T14:20:18.742702+00:00', '--job-id', '55', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/standardized_pipeline.py', '--cfg-path', '/tmp/tmptez8r0o9', '--error-file', '/tmp/tmphqcaa4tu']
[2021-08-07 14:20:20,101] {standard_task_runner.py:77} INFO - Job 55: Subtask create_standardized_table_orders
[2021-08-07 14:20:20,305] {logging_mixin.py:104} INFO - Running <TaskInstance: standardized_pipeline.create_standardized_table_orders 2021-08-07T14:20:18.742702+00:00 [running]> on host ab70d6dafbc6
[2021-08-07 14:20:20,525] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=standardized_pipeline
AIRFLOW_CTX_TASK_ID=create_standardized_table_orders
AIRFLOW_CTX_EXECUTION_DATE=2021-08-07T14:20:18.742702+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-07T14:20:18.742702+00:00
[2021-08-07 14:20:20,539] {mysql.py:70} INFO - Executing: CREATE TABLE IF NOT EXISTS standardized.ORDERS (
  `ORDER_KEY` int NOT NULL,
  `CUSTOMER_KEY` int NOT NULL,
  `ORDER_ORDERSTATUS` text NOT NULL,
  `ORDER_TOTALPRICE` int NOT NULL,
  `ORDER_ORDERDATE` date NOT NULL,
  `ORDER_ORDERPRIORITY` text NOT NULL,
  `ORDER_CLERK` text NOT NULL,
  `ORDER_SHIPPRIORITY` int NOT NULL,
  `ORDER_COMMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
[2021-08-07 14:20:20,566] {base.py:78} INFO - Using connection to: id: dwh_id. Host: dwh, Port: 3306, Schema: , Login: root, Password: ***, extra: {}
[2021-08-07 14:20:20,598] {dbapi.py:204} INFO - Running statement: CREATE TABLE IF NOT EXISTS standardized.ORDERS (
  `ORDER_KEY` int NOT NULL,
  `CUSTOMER_KEY` int NOT NULL,
  `ORDER_ORDERSTATUS` text NOT NULL,
  `ORDER_TOTALPRICE` int NOT NULL,
  `ORDER_ORDERDATE` date NOT NULL,
  `ORDER_ORDERPRIORITY` text NOT NULL,
  `ORDER_CLERK` text NOT NULL,
  `ORDER_SHIPPRIORITY` int NOT NULL,
  `ORDER_COMMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;, parameters: None
[2021-08-07 14:20:20,608] {dbapi.py:212} INFO - Rows affected: 0
[2021-08-07 14:20:20,643] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=standardized_pipeline, task_id=create_standardized_table_orders, execution_date=20210807T142018, start_date=20210807T142019, end_date=20210807T142020
[2021-08-07 14:20:20,829] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-07 14:20:20,866] {local_task_job.py:149} INFO - Task exited with return code 0
